### PR TITLE
ED-3072 Test Case Coverage for Newly Written Code 

### PR DESCRIPTION
**Pull request by sedin-tushar**
Test Case Coverage for Newly Written Code  to Increase Code Coverage of Sonar Cloud

### CODE REVIEW

File: src/app/client/src/app/modules/core/services/form/form.service.ts [hunk 1]
```diff
@@ -57,7 +57,11 @@ export class FormService {
               subType: this.configService.appConfig.formApiTypes[formInputParams.contentType]
               ? this.configService.appConfig.formApiTypes[formInputParams.contentType]
               : formInputParams.contentType,
-              rootOrgId:this.configService.appConfig.frameworkCatConfig.changeChannel ? this.configService.appConfig.frameworkCatConfig.channel : hashTagId ? hashTagId : rootOrgId,
+              rootOrgId: this.configService.appConfig.frameworkCatConfig && this.configService.appConfig.frameworkCatConfig.changeChannel
```
Comment (princegupta1131) [inline src/app/client/src/app/modules/core/services/form/form.service.ts]: @sedin-tushar why you have added this condition this.configService.appConfig.frameworkCatConfig &&

Comment (sedin-tushar) [reply src/app/client/src/app/modules/core/services/form/form.service.ts]: using `this.configService.appConfig.frameworkCatConfig && `-  testcases are passing , using `rootOrgId:this.configService.appConfig.frameworkCatConfig.changeChannel ? ` the test cases are failing due to undefined error 
 

> shall i revert this change and do the appropiate changes in test case and commit ?

Comment (princegupta1131) [reply src/app/client/src/app/modules/core/services/form/form.service.ts]: Please add the fix inside the test case not in the code

Comment (sedin-tushar) [reply src/app/client/src/app/modules/core/services/form/form.service.ts]: Sure , will resolve this

**Pull request merged**